src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback/InnerApi/Requests/GetFeedbackTransactionsToEmailRequest.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+
+namespace SFA.DAS.ApprenticeFeedback.InnerApi.Requests
+{
+    public class GetFeedbackTransactionsToEmailRequest : IGetApiRequest
+    {
+        public int BatchSize { get; set; }
+
+        public GetFeedbackTransactionsToEmailRequest(int batchSize)
+        {
+            BatchSize = batchSize;
+        }
+
+        public string GetUrl => $"api/feedbacktransaction/{BatchSize}";
```

Comment:
I think this may need to be api/feedbacktransaction?batchSize={BatchSize}

Reply:
Ah yes, I'd written it to match my test stub API endpoint, which was also wrong.
Changed to above.